var/spack/repos/builtin/packages/snakemake/package.py

```diff
@@ -14,7 +14,7 @@ class Snakemake(PythonPackage):
 
     version('3.11.2', sha256='f7a3b586bc2195f2dce4a4817b7ec828b6d2a0cff74a04e0f7566dcd923f9761')
 
-    depends_on('python@3.3:')
+    depends_on('python@3.3:3.6.999')
```

Comment:
I guess this works for now, but needs to be updated as soon as we'll get more recent versions of this package.